drivers/storage/portworx/testspec/autopilotDeployment.yaml

```diff
@@ -75,6 +82,15 @@ spec:
                     operator: Exists
                   - key: node-role.kubernetes.io/worker
                     operator: Exists
+              - matchExpressions:
+                  - key: px/enabled
+                    operator: NotIn
+                    values:
+                      - "false"
+                  - key: node-role.kubernetes.io/control-plane
+                    operator: Exists
+                  - key: node-role.kubernetes.io/worker
+                    operator: Exists
```

Comment:
Are these rules correct?

It seems like we are looking for..
* non-master nodes (workers?) that are NOT tagged as `control-plane`, or
* worker nodes that ARE tagged as `control-plane`

Reply:
The k8s master node labels was changed from `../master` to `../control-plane`. This is basically covering the case where a master/control-plane node is also a worker node and we need to support it and treat it like a worker node.

Reply:
Would it make sense to simplify the rules, e.g.
1. if `worker`  (that's essentially combined rule#2 and rule#3), or
2. if `NOT master and NOT control-plane` (that's rule#1)